src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -168,7 +171,7 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook, Path filePath)
 
         // Triggers the saveAddressBook method by executing an add command
         String addCommand = AddCommand.COMMAND_WORD + NAME_DESC_AMY + PHONE_DESC_AMY
-                + INCOME_DESC_AMY + EMAIL_DESC_AMY + ADDRESS_DESC_AMY + FAMILY_DESC_AMY;
+                + INCOME_DESC_AMY + EMAIL_DESC_AMY + ADDRESS_DESC_AMY + FAMILY_DESC_AMY + TAG_DESC_AMY;
         Person expectedPerson = new PersonBuilder(AMY).withTags().build();
```

Comment:
`import static seedu.address.logic.commands.CommandTestUtil.VALID_TAG_AMY;`

```suggestion
        Person expectedPerson = new PersonBuilder(AMY).withTags(VALID_TAG_AMY).build();
```

Since its compulsory